.travis.yml

```diff
@@ -37,5 +37,6 @@ script:
     - coverage xml -i
     - python-codacy-coverage -r coverage.xml
     - codecov
+    - mypy --ignore-missing-imports --show-error-codes .
```

Comment:
I'm going to add `pytest-mypy` to requirements_dev, and run the mypy check through pytest along with the coverage check. I'll also add a mypy.ini skipping import errors.